website/docs/advanced/instantiate_objects.md

```diff
@@ -0,0 +1,141 @@
+---
+id: instantiate
+title: Instantiate objects and calling functions
+sidebar_label: Instantiate objects and calling functions
+---
+
+Hydra provides `hydra.utils.call()` (and its alias `hydra.utils.instantiate()`) for instantiating objects and calling functions While `instantiate` is an alias for `call`, you may prefer to use `call` for invoking functions and class methods, and `instantiate` for creating objects.	
+
+```python
+def call(config: Union[ObjectConf, DictConfig], *args: Any, **kwargs: Any) -> Any:
+    """
+    :param config: An ObjectConf or DictConfig describing what to call and what params to use
+    :param args: optional positional parameters pass-through
+    :param kwargs: optional named parameters pass-through
+    :return: the return value from the specified method
+    """
+```
+
+```python
+def instantiate(config: Union[ObjectConf, DictConfig], *args: Any, **kwargs: Any) -> Any:
+    """
+    :param config: An ObjectConf or DictConfig describing what to instantiate and what params to use
+    :param args: optional positional parameters pass-through
+    :param kwargs: optional named parameters pass-through
+    :return: the return value from the specified class
+    """
+```
```

Comment:
I think no point in documenting both.
include instantiate and mention that call is available as an alias.

Reply:
Sounds good. Probably it doesnt matter but earlier documentation always mentioned `call` (as the primary method) and `instantiate` as the alias (I know no real difference). I will stick to that convention.